common/djangoapps/entitlements/api/v1/urls.py

```diff
@@ -1,11 +1,22 @@
 from django.conf.urls import url, include
 from rest_framework.routers import DefaultRouter
 
-from .views import EntitlementViewSet
+from .views import EntitlementViewSet, EntitlementEnrollmentViewSet
 
 router = DefaultRouter()
 router.register(r'entitlements', EntitlementViewSet, base_name='entitlements')
 
+ENROLLMENTS_VIEW = EntitlementEnrollmentViewSet.as_view({
+    'post': 'create',
+    'delete': 'destroy',
+})
+
+
 urlpatterns = [
     url(r'', include(router.urls)),
+    url(
+        r'entitlements/(?P<uuid>[0-9a-f-]+)/enrollments/$',
```

Comment:
This regex doesn't reflect an actual UUID, you can just as easily have something like `entitlements/---/enrollments/ evaluate to true in here. 

Separate thought, is it worth enforcing the trailing slash?